challenge_10/rack_response.rb

```diff
@@ -58,9 +61,11 @@ def set_up_blogs_store(store_path)
 
     # Prepare response
     response.content_type = 'text/html'
+    response.finish
```

Comment:
I suppose this is an example of a place where the combination of the `Rack::Response` block constructor and the pre-v2.1.0 behaviour was kind of convenient: if we didn’t care about explicitly calling `#finish` (i.e. to pass it a block so we could send a synchronous response), we could move all our response-generation logic inside `Rack::Response.new do |response| … end` and allow the response object to be returned from that, then let the application server implicitly call `#finish` (via `#to_ary`) when it tried to treat the response as an array. Instead we now have to be explicit about calling `#finish` and we don’t return the response at all, which in turn makes the block constructor form less elegant.

Reply:
That makes a lot of sense! I noticed in [Aaron's PR](https://github.com/rack/rack/pull/1453) that the argument was "Responses are not arrays, so we should not allow them to be implicitly coerced to an array.", which makes sense too. It is convenient, but a little.. sneaky to have a response object get magically coerced into an array 😄